server/services/hospitalSearchService.ts

```diff
@@ -0,0 +1,33 @@
+import PrisonApiClient, { Prison } from '../data/prisonApiClient'
+import { User } from '../data/hmppsAuthClient'
+
+export type Hospital = {
+  agencyId: string
+  description: string
+  longDescription: string
+  agencyType: string
+  active: true
+}
+
+export default class HospitalSearchService {
+  async getHospitals(user: User): Promise<Prison[]> {
+    // agencies endpoint just needs a valid token without any special roles required
+    const client = new PrisonApiClient(user.token)
```

Comment:
note I have switched the code to use the user token here as the Prison API `/agencies` endpoint doesn't need any roles, only a valid token